src/error/errorHandler.ts

```diff
@@ -0,0 +1,77 @@
+import { Context } from '@azure/functions';
+
+import { Options } from '../middleware';
+import { ApplicationError } from './ApplicationError';
+
+type ErrorWithMessage = {
+    message: string;
+    stack?: string;
+};
+
+const isErrorWithMessage = (error: unknown): error is ErrorWithMessage => {
+    return (
+        typeof error === 'object' &&
+        error !== null &&
+        'message' in error &&
+        typeof (error as Record<string, unknown>).message === 'string'
+    );
+};
+
+const logErrorObject = (error: object | null, context: Context) => {
+    if (error === null) {
+        context.log.error('The provided error was eq to null - unable to log a specific error-message');
+        return;
+    }
+
+    if (isErrorWithMessage(error)) {
+        context.log.error({ message: error.message, stack: error.stack });
+    } else {
+        try {
+            const errorAsJson = JSON.stringify(error);
+            if (errorAsJson === '{}') {
+                context.log.error(error.toString());
+            }
```

Comment:
When the error is something unparsable like `blablub`, the error ist logged with `toString()`. In case the error is a valid JSON without a "message" part, nothing is logged ? Or did I miss something ?

Reply:
Yeah, my bad. I will fix it.

Reply:
Fixed.